app/models/load_balancer_pool_member.rb

```diff
@@ -1,7 +1,30 @@
 class LoadBalancerPoolMember < ApplicationRecord
-  belongs_to :load_balancer_pool
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::NetworkManager"
   belongs_to :cloud_tenant
   belongs_to :resource_group
```

Comment:
This isn't a change in this PR, but probably [same](https://github.com/ManageIQ/manageiq/pull/10404/files#r74667921).